Jenkinsfile

```diff
@@ -585,15 +580,18 @@ def runAllScopes(Map args = [:]) {
 def runTest(Map args = [:]){
   def stack = args.stack
   def scope = args.scope
+  def allocateNode = args.get('allocateNode', false)
   def mode = env.MODE == 'none' ? 'Puppeteer' : env.MODE
   def stackVersion = (stack == '7.x') ? artifactsApi(action: '7.x-version') : stack
+
   withGithubNotify(context: "Test ${scope} - ${stack} - ${mode}", tab: 'tests') {
-    runScript(
-      label: "${scope}",
-      stack: "${stackVersion}",
-      scope: "${scope}",
-      goal: 'test'
-    )
+    if (allocateNode) {
+      withNode(labels: 'linux && immutable', sleepMax: 5, forceWorspace: true, forceWorker: true){
```

Comment:
It requests the ephemeral worker with an unique ID (that's something the `Gobld` provisioner does under the hood)

<img width="1441" alt="image" src="https://user-images.githubusercontent.com/2871786/163824478-b2bb8225-5c96-4a9a-83f1-d38bdb615c25.png">